src/core/stdc/stdlib.d

```diff
@@ -153,6 +153,13 @@ else
     void    srand(uint seed);
 }
 
+version (CRuntime_Glibc)
+{
+    version = AlignedAllocSupported;
+}
+// Add other C runtimes as necessary...
+else {}
```

Comment:
Could be a bit more elegant:

```suggestion
version (CRuntime_Glibc)
    version = AlignedAllocSupported;
else {}
```

(The comment was just for the discussion in this PR, I think it's out of place in the code.)